packages/destination-actions/src/destinations/drip/identify/index.ts

```diff
@@ -0,0 +1,111 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const person = (payload: Payload) => {
+  return {
+    custom_fields: payload.customFields,
+    email: payload.email,
+    ip_address: payload.ip,
+    new_email: payload.newEmail,
+    sms_number: payload.sms,
+    status: payload.status,
+    status_updated_at: payload.statusUpdatedAt,
+    tags: payload.tags,
+    time_zone: payload.timezone
+    // prospect: true // omit for now, lead scoring is not promoted by our product
+  }
+}
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Identify',
+  description: 'Identify person in Drip',
+  fields: {
+    customFields: {
+      description: "Custom fields to add to a person's profile.",
+      label: 'Custom fields',
+      required: false,
+      type: 'object', // dictionary of strings to strings
+      default: { '@path': '$.properties.customFields' }
+    },
+    email: {
+      description: "The person's email address.",
+      label: 'Email Address',
+      required: true,
+      type: 'string',
+      format: 'email',
+      default: { '@path': '$.traits.email' }
+    },
+    ip: {
+      description: "The person's ip address.",
+      label: 'IP Address',
+      required: false,
+      type: 'string',
+      format: 'ipv4',
+      default: { '@path': '$.context.ip' }
+    },
+    newEmail: {
+      description: "The person's new email address.",
+      label: 'New Email Address',
+      required: false,
+      type: 'string',
+      format: 'email',
+      default: { '@path': '$.traits.newEmail' }
+    },
```

Comment:
I see you added this and I don't disagree. I can't help speculating on what the future might actually look like, since I want to avoid any one-way doors.

I think in the future we might want to pass an id type, and maybe even an id. So `email` would mean change this person's email to X, unless it the id type was `email`.

I think this model could be to far out to consider planning for though 🤔

Reply:
Are you concerned that without an ID, future events using an old email would get dropped?

I suppose email changes probably aren't a huge slice of customers' event pie, so it might make more sense to not include newEmail and avoid the confusion altogether. 

Would there any worry for added complexity with adding an ID type?

Reply:
I just went ahead and removed it. We can always add it in later.